src/app/Poll.php

```diff
@@ -13,21 +13,29 @@ class Poll extends Model
     /**
      * Available choices for this poll
      */
-    public function choices() {
+    public function choices()
+    {
         return $this->hasMany('App\Choice');
     }
 
     /**
      * List of users that answered to this poll
      */
-    public function users() {
+    public function users()
+    {
         return $this->belongsToMany('App\User')->withPivot('choice_id');
     }
 
     /**
      * Linked presentation
      */
-    public function presentation() {
+    public function presentation()
+    {
         return $this->belongsTo('App\Presentation');
     }
+
+    public function presenters()
+    {
+        return $this->presentation->users()->where('role', 'presenter')->get();
```

Comment:
you should call a method `presenters` on `Presentation` as it's the job of the class `Presentation` to know who are the presenters
```php
return $this->presentation->presenters();
```

Reply:
I don't find any such function in `Presentation`, did you mean `moderators` as per below ?

Reply:
You're allowed to create new functions if needed

Reply:
Fixed as per 734eedb